src/libs/actions/TeachersUnite.js

```diff
@@ -0,0 +1,175 @@
+import _ from 'underscore';
+import Onyx from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import ONYXKEYS from '../../ONYXKEYS';
+import * as API from '../API';
+import Navigation from '../Navigation/Navigation';
+import CONST from '../../CONST';
+import * as ReportUtils from '../ReportUtils';
+import * as OptionsListUtils from '../OptionsListUtils';
+
+let sessionEmail = '';
+let sessionAccountID = 0;
+Onyx.connect({
+    key: ONYXKEYS.SESSION,
+    callback: (val) => {
+        sessionEmail = lodashGet(val, 'email', '');
+        sessionAccountID = lodashGet(val, 'accountID', 0);
+    },
+});
+
+let allPersonalDetails;
+Onyx.connect({
+    key: ONYXKEYS.PERSONAL_DETAILS_LIST,
+    callback: (val) => (allPersonalDetails = val),
+});
+
+/**
+ * @param {String} firstName
+ * @param {String} phoneOrEmail
```

Comment:
Hmm we should receive both (email and phone number), the problem is that the API is expecting the parameter `email` even though it can also be a number (which is weird). 

Let's change it to `partnerUserID` instead in both commands, I will need to change the API command, but I think it will be best to avoid either `email` or `phoneOrEmail` names as parameter.